tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -208,12 +208,12 @@ private async Task<AuthenticationResult> RunAtsAsync(IPublicClientApplication pc
 
             if (!string.IsNullOrEmpty(loginHint))
             {
-                if (IsMsaPassthroughConfigured())
-                {
-                    // TODO: bogavril - move this exception in WAM
-                    throw new InvalidOperationException(
-                        "[TEST APP FAILURE] Do not use login hint on AcquireTokenSilent for MSA-Passthrough. Use the IAccount overload.");
-                }
+                //if (IsMsaPassthroughConfigured())
```

Comment:
This check/error can be removed then right?

Reply:
removed